ethcore/spec/src/spec.rs

```diff
@@ -426,7 +427,17 @@ impl Spec {
 			ethjson::spec::Engine::Null(null) => Arc::new(NullEngine::new(null.params.into(), machine)),
 			ethjson::spec::Engine::Ethash(ethash) => {
 				// Specific transitions for Ethash-based networks
-				for block in &[ethash.params.homestead_transition, ethash.params.dao_hardfork_transition] {
+				for block in &[
+					ethash.params.homestead_transition,
+					ethash.params.dao_hardfork_transition,
+					ethash.params.difficulty_hardfork_transition,
+					ethash.params.bomb_defuse_transition,
+					ethash.params.eip100b_transition,
+					ethash.params.ecip1010_pause_transition,
+					ethash.params.ecip1010_continue_transition,
+					ethash.params.ecip1017_era_rounds,
+					ethash.params.expip2_transition,
```

Comment:
Looks like a typo there?

exip instead of ecip

Reply:
whelps, looks like i misread it. sorry!

Reply:
No typo here. It's `EXPIP-2` related to Expanse chain, not Classic.